network/dns_packet.ksy

```diff
@@ -89,7 +94,7 @@ types:
         size: length
     instances:
       is_pointer:
-        value: length == 0b1100_0000
+        value: length >= 192
```

Comment:
Can you explain why this change was necessary?

Reply:
https://tools.ietf.org/html/rfc1035
Offset can be non zero:

```
    +--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+
    | 1  1|                OFFSET                   |
    +--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+
```